### PR TITLE
Adding the new peer review guide to the landing page

### PR DESCRIPTION
**Pull request by bergerhoffer**

### CODE REVIEW

**Comment by bergerhoffer** [comment]
@ncbaratta @sterobin @rkratky Can any of you please review? This is adding the new CCS peer review guide to the landing page.

**Comment by bergerhoffer** [comment]
FYI @leswilliams44

**Review by leswilliams44 — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -115,6 +115,12 @@ <h2 id="modular-documentation-project">Modular Documentation Project</h2>
 
   <p>Modular Documentation Project provides guidelines and examples for writing technical documentation using a modular framework.</p>
 
+  <h2 id="peer-review">Red Hat peer review guide for technical documentation</h2>
+
+  <p><a href="https://redhat-documentation.github.io/peer-review/">Red Hat peer review guide for technical documentation</a></p>
+
+  <p>This guide provides information about best practices for peer reviewing Red Hat technical documentation.</p>
```
Comment (sterobin) [inline index.html]:
```suggestion
  <p>This guide provides guidelines for peer reviewing Red Hat technical documentation.</p>
```

To simplify and to be consistent with the other descriptions.

**Review by sterobin — COMMENTED** [review]
@bergerhoffer Looks great, just a trivial comment.

File: index.html (hunk 1 above)
Comment (bergerhoffer) [reply index.html]: Good suggestion thanks @sterobin, updated! I had grabbed this from the peer review guide itself, so I'll look to update it there as well.

**Review by sterobin — APPROVED** [review]

**Pull request merged**